public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -336,47 +338,47 @@ include ../../../../_includes/_util-fns
 
 
   What if it had a dependency? What if it reported its activities through a logging service?
-  We'd apply the same *constructor injection* pattern, 
-  adding a constructor that takes a `logger` parameter.
+  We'd apply the same *constructor injection* pattern,
+  adding a constructor that takes a `Logger` parameter.
 
   Here is the revision compared to the original.
-  
+
 +makeTabs(
-  `dependency-injection/ts/app/heroes/hero.service.2.ts, 
+  `dependency-injection/ts/app/heroes/hero.service.2.ts,
   dependency-injection/ts/app/heroes/hero.service.1.ts`,
-  null, 
+  null,
   `app/heroes/hero.service (v.2),
   app/heroes/hero.service (v.1)`)
 
 :marked
   The constructor now asks for an injected instance of a `Logger` and stores it in a private property called `_logger`.
-  We call that property within our `getHeroes()` method when anyone asks for heroes.
+  We call that property within our `getHeroes` method when anyone asks for heroes.
 
   <a id="injectable"></a>
-  ### Why *@Injectable*?
+  ### Why @Injectable?
```

Comment:
Is it really that important to have different fonts in headers?

Most places I've worked, we use only plain old bold font in headers (no code font, italics, or other special cases). Some of that might have been due to tool limitations, but it might have been a purely esthetic decision.

In this particular case, the @ and the capital I make this unmistakably a decorator. And I tend to thing of italics as variables (e.g. <code>cp <em>file1 file2</em></code>).

All that said, I don't care too strongly about this. It's my knee-jerk reaction to avoid italics in headings and question them elsewhere... but I can control my knee.